documentation/userguide.md

```diff
@@ -205,7 +205,7 @@ require special care when trying to use backlashes since these are used as an es
 circumstances.
 
 There is a third selector `loadedAssemblyPathRegex` that is currently only used for configuring trace upload based on the assemblies that are loaded.
-See example regarding "multiple applications that are started from the same process" at the end of the documentation for details.
+[See example](## Example: Profile and upload multiple applications that are started from the same process) regarding "multiple applications that are started from the same process" at the end of the documentation for details.
```

Comment:
It looks like GitHub couldn't nicely render the link. 😞 Revert?

![image](https://github.com/cqse/teamscale-profiler-dotnet/assets/17203922/9a3dd7bd-92b8-4cd7-82aa-0d1048fdac73)